addons/medical_treatment/functions/fnc_medicationLocal.sqf

```diff
@@ -59,6 +59,7 @@ private _painReduce             = GET_NUMBER(_medicationConfig >> "painReduce",g
 private _timeInSystem           = GET_NUMBER(_medicationConfig >> "timeInSystem",getNumber (_defaultConfig >> "timeInSystem"));
 private _timeTillMaxEffect      = GET_NUMBER(_medicationConfig >> "timeTillMaxEffect",getNumber (_defaultConfig >> "timeTillMaxEffect"));
 private _maxDose                = GET_NUMBER(_medicationConfig >> "maxDose",getNumber (_defaultConfig >> "maxDose"));
+private _maxDose                = GET_NUMBER(_medicationConfig >> "riskDose",getNumber (_defaultConfig >> "riskDose"));
```

Comment:
```suggestion
private _riskDose               = GET_NUMBER(_medicationConfig >> "riskDose",getNumber (_defaultConfig >> "riskDose"));
```

I don't like the name `riskDose` though, it's not clear to me.
The problem is that atm I don't have a better name, need to think on it.

Reply:
Same

Reply:
MaxDose is fine, it's the max before problems can start.

Reply:
When in doubt, be verbose: `_maxSafeDose`

Reply:
The git suggestion is to fix a double declaration of `_maxDose`.

Separately from that, I don't like `riskDose`.

Reply:
`maxDose` -> `maxSafeDose`
`riskDose`  -> `chanceDoses`

Reply:
`maxDose` -> `maxSafeDose`
`riskDose`  -> `chanceDoses`